system-tests/test-binary/node_versions_spec.ts

```diff
@@ -26,17 +26,15 @@ function smokeTestDockerImage (dockerImage: string) {
 
 describe('binary node versions', () => {
   [
-    'cypress/base:18.16.1',
-    'cypress/base:20.12.2',
+    'cypress/base:20.18.0',
     'cypress/base:22.0.0',
     'cypress/base:22.7.0',
   ].forEach(smokeTestDockerImage)
 })
 
 describe('type: module', () => {
   [
-    'cypress/base:18.16.1',
-    'cypress/base:20.12.2',
+    'cypress/base:20.18.0',
```

Comment:
I think we still need the `18.16.1` tests here and above as users can still run cypress with node 18. Unless that breaks?

Reply:
Good catch. I think I originally thought we'd drop support, but 18 is still in [maintenance mode](https://nodejs.org/en/about/previous-releases), so for now I think we can keep supporting it.

Reply:
```suggestion
    'cypress/base:18.16.1',
    'cypress/base:20.12.2',
    'cypress/base:20.18.0',
```